### ISSUE TITLE
Vanilla

### ISSUE THREAD

**Issue by solomonshalom** [stackblitz/core#2038]
I wanted to open a vanilla HTML project but couldn't as it didn't contain a package.json, this is really irritating :(

**Comment by AleksandrSl** [comment]
Hi, @solomonshalom could you give more details? Maybe you have repository with your project?
Our vanilla HTML projects should work without any `package.json`, [example](https://stackblitz.com/edit/web-platform-vbxaks?file=index.html)

**Comment by davidvandenbor** [comment]
I've noticed the same problem. 🙂 I think solomonshalom is referring to this: if you try to open a vanilla HTML github repo in StackBlitz by using the URL method, the import crashes. Because StackBlitz can only render the project if it finds a package.json file in the root directory of the GitHub repository!

https://stackblitz.com/github//davidvandenbor/html-starter-template

The issue is: StackBlitz doesn't specify what the content of the JSON file should be. I made my own JSON file to trigger a Parcel build in StackBlitz. But the Parcel build really taxes the MacBooks processor (the fan makes a lot of noise, which isn't the case with the standard StackBlitz vanilla HTML build, which doesn't tax the processor)
